packages/foundry/script/VerifyAll.s.sol

```diff
@@ -0,0 +1,104 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import "forge-std/Vm.sol";
+
+contract VerifyAll is Script {
+    uint96 currTransactionIdx;
+
+    function run() external {
+        string memory root = vm.projectRoot();
+        string memory path = string.concat(
+            root,
+            "/broadcast/Deploy.s.sol/80001/run-latest.json"
```

Comment:
I tried verifying contract which was deployed to sepolia and it gave me : 

![Screenshot 2023-08-15 at 9 10 01 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/9246b159-4fef-4f10-b64f-e6aa25a39dcb)

I think its because of we have hardcoded `80001` ?